### ISSUE TITLE
[iOS] Touch on slider bar brings the thumb to beginning of the bar

### ISSUE THREAD

**Issue by ludovic-gonthier** [react-native-community/react-native-slider#134]
## Environment

```
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
    Memory: 1.36 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.11.0 - ~/.nvm/versions/node/v10.11.0/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.4.1 - ~/.nvm/versions/node/v10.11.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
    Android SDK:
      API Levels: 28
      Build Tools: 28.0.3
      System Images: android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5900203
    Xcode: 10.3/10G8 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.1 => 0.61.1
```

## Description

On iOS, when I try to touch the slide to go increase/decrease its value, the thumb always goes to the beginning of the slider without triggering the `onValueChange` callback.
On Android, this behavior works fine.

## Reproducible Demo

```jsx
<Slider
  minimumValue={6}
  value={16}
  maximumValue={26}
  onValueChange={(value) => console.log('Value changed: ', value}
  step={1}
/>
```

**Comment by mcousillas6** [comment]
I'm having the same issue, is there any way to disable the tap to seek until this is resolved?

**Comment by BunlongOung** [comment]
@mcousillas6 You can downgrade to use `2.0.3` for now.

**Comment by BunlongOung** [comment]
I'm facing this issue, too. Downgrade to 2.0.3 can solve the problem for now since it does not handle touch on slide bar

**Comment by michalchudziak** [comment]
I guess it'd be best if we add an option to toggle that behavior via props. Anyone interested in sending a PR? cc @wojteg1337

**Comment by mcousillas6** [comment]
> I guess it'd be best if we add an option to toggle that behavior via props. Anyone interested in sending a PR? cc @wojteg1337

I can try and tackle this next week if you guys want

**Comment by michalchudziak** [comment]
@mcousillas6 Actually, I found a simple solution, and I'll quickfix that now, as it seems like high prio. Thank you for your engagement, and we'd love to have some help with other issues if you'd like to contribute :)

**Comment by michalchudziak** [comment]
This issue should be fixed in version `2.0.6`. Please check it out and re-open this if it still occurs :)

**Comment by SiyabongaNzulwana** [comment]
@michalchudziak what if I am still using the react native ` "react-native": "0.59.9"` ? How do I get access to that feature you built, I really need it.
